image/discourse_bench/Dockerfile

```diff
@@ -17,11 +19,12 @@ RUN gem install facter &&\
 
 # reconfigure PostgreSQL template databases to use UTF-8
 # otherwise rake db:create fails
-RUN pg_dropcluster --stop 9.3 main &&\
-    pg_createcluster --start -e UTF-8 9.3 main
+RUN pg_dropcluster --stop 9.5 main &&\
```

Comment:
Got it!  I'll have a PR into discourse to tweak `scripts/bench.rb` so it stops attempting to load unused external facts.  With that change in place the benchmark runs successfully on my machine (via discourse/discourse_bench):
```
Your Results: (note for timings- percentile is first, duration is second in millisecs)
---
categories_admin:
  50: 94
  75: 100
  90: 108
  99: 130
home_admin:
  50: 98
  75: 104
  90: 115
  99: 145
topic_admin:
  50: 49
  75: 50
  90: 53
  99: 71
categories:
  50: 52
  75: 55
  90: 60
  99: 78
home:
  50: 60
  75: 63
  90: 73
  99: 94
topic:
  50: 18
  75: 18
  90: 19
  99: 35
timings:
  load_rails: 5266
ruby-version: 2.3.1-p112
rss_kb: 269892
pss_kb: 261193
operatingsystem: Ubuntu
virtual: docker
architecture: amd64
physicalprocessorcount: 4
kernelversion: 4.4.20
processor0: Intel(R) Core(TM) i7-4850HQ CPU @ 2.30GHz
memorysize: 1.95 GB
```